dict_test.go

```
@@ -14,887 +14,938 @@ import (
-	_ = r.Close()
-	// Raw dict (ID 0) doesn't embed a dict ID in the frame, so the error
-	// is a corruption from bad offsets rather than ErrUnknownDictionary.
-	if !errors.Is(err, &ErrCorrupted{}) {
-		t.Fatalf("raw dict after clear: expected ErrCorrupted, got: %v", err)
-	}
+		dec.SetRawDict(dict2)
+		if err := r.Reset(bytes.NewReader(frame2)); err != nil {
+			t.Fatal(err)
+		}
+		got2, err := io.ReadAll(r)
+		r.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `r.Close` is not checked

(errcheck)